src/config/constants/index.ts

```diff
@@ -185,12 +187,12 @@ export const getConstants = memoize((): Constants => {
     SUPPORTED_LANGS: ['en', 'es', 'ja', 'zh'],
     DUST_AMOUNT_USD: '10000000',
     ASSETS_ICON_URL: 'https://raw.githubusercontent.com/yearn/yearn-assets/master/icons/multichain-tokens/1/',
-    Arbiter_GOERLI: '0xde8f0f6769284e41bf0f82d0545141c15a3e4ad1',
-    Oracle_GOERLI: '0x7233038e589913dca4b6d15ff8bba263a433aed2',
-    SwapTarget_GOERLI: '0xcb7b9188ada88cb0c991c807acc6b44097059dec',
-    LineFactory_GOERLI: '0xc23b896F2b4aE3E6362B0D536113Fa2F0C9b8886',
-    SecuredLine_GOERLI: '0x32cD4087c98C09A89Dd5c45965FB13ED64c45456',
-    KibaSero_oracle: '0x0B3807b858B5fa24a39bced436DBc5A988Ca58d6',
+    //Arbiter_GOERLI: '0xde8f0f6769284e41bf0f82d0545141c15a3e4ad1',
+    //Oracle_GOERLI: '0x7233038e589913dca4b6d15ff8bba263a433aed2',
+    //SwapTarget_GOERLI: '0xcb7b9188ada88cb0c991c807acc6b44097059dec',
+    LineFactory_GOERLI: networks.goerli.lineFactory,
```

Comment:
can u make it all upper case like the rest of the vars pleaase. And delete all the unused ones.

Reply:
Done